compiler/passes/scopeResolve.cpp

```diff
@@ -1922,8 +1922,8 @@ static void resolveEnumeratedTypes() {
         if (EnumType* type = toEnumType(first->var->type)) {
           if (SymExpr* second = toSymExpr(call->get(2))) {
             const char* name;
-            bool found = false;
-
+            //bool found = false;
+            /*
```

Comment:
I think removing them is reasonable - but if the changes I suggested earlier mean we don't need to remove them (or need to still have them), then it makes reversing that change much easier